.nsprc

```diff
@@ -1,4 +1,3 @@
 {
-  // https://github.com/nodesecurity/nsp#exceptions
```

Comment:
We can't use comments anymore. Boo. https://github.com/nodesecurity/nsp/issues/207